composer.lock

```diff
@@ -3513,16 +3513,16 @@
         },
         {
             "name": "drupal/core",
-            "version": "9.3.0",
+            "version": "9.3.2",
```

Comment:
@Rajeshreeputra Why it's updating Drupal core as it's already done with other ticket ?
cc: @chandan-singh7929

Reply:
I think rebase is required here, and should be fine.

Reply:
Rebase completed and changes pushed to origin.